core/pom.xml

```diff
@@ -267,6 +269,31 @@
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>flatten-maven-plugin</artifactId>
```

Comment:
@thegreystone 
Can the flatten-maven-plugin also be used for jmc too? The artifacts produced for jmc-core look good now, but the ones produced for jmc have m2 pom files with ${revision}${changelist}.

Reply:
Yes, certainly. Added. Let me know if it works the way you want.

Reply:
This was added to the `pluginManagement` section so it will configure plugins for projects down the tree but not actually be active. I think it should be moved to the `plugins` section one level up within `build`.